app/models/studio.rb

```diff
@@ -1,3 +1,11 @@
 class Studio < ApplicationRecord
+  validates_presence_of :name, :location
+
   has_many :movies
+  has_many :movie_actors, through: :movies
+  has_many :actors, through: :movie_actors
```

Comment:
This is great! You can also just add one additional association instead of 2 to achieve the same outcome -- because we've already set up the shortcut between movies and actors in the other models, we can just do:
`has_many :actors, through: :movies` without including the joins table